R/render_report.R

```diff
@@ -1,4 +1,4 @@
-#' Render Report
+#' Render Summary Report
 #'
 #' @param output an output file path
 #' @param prams a string "ask", which will launch an interactive window to enter
```

Comment:
The argument seems to be 'params', instead of 'param'

Reply:
Isn't it param a standard variable in here?